scotty

```
@@ -86,15 +86,19 @@ def set_env(env="base"):
             os.environ[key] = val
 
 
-def run_commands(commands):
+def run_commands(commands, exit_on_error=True):
```

are you sure this is the default you want everwhere

---

I don't think it's best to exit here, instead return whether or not it failed and it should be handled outside of this command. Maybe throw an exception?

---

This allows for the behavior that was already present to occur